### ISSUE TITLE
About the experimental results

### ISSUE THREAD

**Issue by Cappuccino-Sugar** [moen-hyb/ATMOL#2]
Hello, it is a pleasure to read your paper, the result is shocking. But there are still some questions : 
  is it true that the pre-trained model was only trained for 20 epochs? 
  how many epochs were trained for the downstream task? 
  is the result of the downstream task a validation set or a test set? I ran your code and could not surface your AUC!
  what is your approximate GPU memory usage during pre-training, I only have 30% here
I hope you can answer these questions, thanks!

**Comment by Duckbluee** [comment]
Hi, I also want to duplicate the results, but I even can't run the code for pre-training successfully, there are many errors. 
Especially here, 
'x, edge_index, batch, x_size, edge_size = data1.x, data1.edge_index,data1.batch, data1.x_size, data1.edge_size' 
in model_gat_pre.py.
And the the error is about format of attributes, I worry that making too many changes will affect code integrity.
So can you tell me how you overcome this problem or  what changes you made before you successfully run the code.
Thanks!
